src/tools/css/get-css.js

```diff
@@ -171,7 +171,10 @@ const getCSS = (fondue) => {
 		" */\n\n";
 
 	stylesheet = stylesheet + stylesheetIntro;
-	stylesheet = stylesheet + getFontFace(fondue);
+
+	if (!exclude.includes("font-face")) {
```

Comment:
@RoelN I think this mechanism is fine, if there aren't too many options in the future. There are some other patterns you could consider though.

1. Don't immediately return a CSS string, but return an object describing the CSS (like an AST, but perhaps a bit more high level):

```js
const getCSS = function(fondue) {
  return [
    { element: "font-face", css: "..." },
    { element: "variable", css: "..." }
  ];
};
```

Then the consumer can decide how to render this and which parts to include. For convenience you could provide a default renderer that concatenates all parts, like the current implementation is doing.

Advantage is that this function will remain simple and focused on producing the CSS and externally you can control what to render.

2. Work with a render callback

This would be pretty similar to option 1, but is "streaming" i.e. you don't have to build up the entire CSS and return that as a single object. Instead you'll pass each CSS section to the render callback as soon as it's produced.

```js
const getCSS = function(fondue, render) {
  render("font-face", fontFaceCSS);
  // ...
  render("variable", variableCSS);
};

// consumer
let output = '';
getCSS(fondue, function(section, css) {
  if(section != 'font-face') output += css;
});
```

For any option I would make the library export constants, to not have to rely on magic strings like `font-face`. So something like:

```js
export const ELEMENT_FONT_FACE = "font-face";

// consumer

import { ELEMENT_FONT_FACE } from "@wakamai-fondue/engine/css";
```

WDYT?